components/suggest/src/schema.rs

```diff
@@ -92,6 +92,11 @@ pub const SQL: &str = "
         record_id TEXT NOT NULL
     ) WITHOUT ROWID;
 
+    CREATE TABLE yelp_custom_details(
```

Comment:
Unlike other suggestions, we build Yelp suggestions on the fly, so there is no records in the `suggestions` table for Yelp. Thus, we don't have the `suggestion_id` in this table. Also, there should be only one record in this table, which feels odd. As discussed in another thread, maybe we can move the `icon_id` to the `suggestion_providers` table in the future.